src/EventStore/Broadway/Drivers/DynamoDb.php

```diff
@@ -0,0 +1,49 @@
+<?php namespace Nwidart\LaravelBroadway\EventStore\Broadway\Drivers;
+
+use Aws\DynamoDb\DynamoDbClient;
+use Broadway\EventStore\DynamoDb\DynamoDbEventStore;
+use Nwidart\LaravelBroadway\EventStore\Driver;
+use Aws\Credentials\Credentials;
+
+class DynamoDb implements Driver
+{
+    /**
+     * @var \Illuminate\Config\Repository
+     */
+    private $config;
+
+    public function __construct()
+    {
+        $this->config = app(\Illuminate\Config\Repository::class);
+    }
+
+    /**
+     * @return object
+     */
+    public function getDriver()
+    {
+        $payloadSerializer = app(\Broadway\Serializer\Serializer::class);
+        $metadataSerializer = app(\Broadway\Serializer\Serializer::class);
+
+        $table = $this->config->get('broadway.event-store.table', 'event_store');
+        $endpoint =  $this->config->get('broadway.event-store.endpoint', '');
+        $accessKeyId =  $this->config->get('broadway.event-store.access_key_id', '');
+        $secretAccessKey =  $this->config->get('broadway.event-store.secret_access_key', '');
+        $region =  $this->config->get('broadway.event-store.region', '');
+
+        $credentials = new Credentials($accessKeyId, $secretAccessKey);
```

Comment:
My only issue around this is where in my company we don't configure our applications with AWS access / secret keys. We use [EC2 IAM Roles](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html), which allows the applications to pull out secret keys dynamically from the EC2 Metadata service.

Reply:
Yeah, I thought to integrate another access manager where you can use secert keys or EC2 IAM Roles, it could be a new next feature @Lavoaster ?